tests/integration/targets/nxos_ospfv3/tests/common/merged.yaml

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
Do we have the test for empty config?

Reply:
Since this check is handled by the RM Base Engine now, it probably makes more sense to add a test for that code. But since that is not in place right now, I'll add it for this module.